core/jazz_services/config/prod-config.json

```diff
@@ -1,8 +1,8 @@
 {
     "services_table": "{inst_stack_prefix}_services_prod",
-    "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status"],
+    "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status", "deployment_accounts"],
     "service_update_fields": ["description", "email", "slack_channel", "tags", "repository", "status","metadata","is_public_endpoint"],
-    "service_filter_params": ["service", "domain", "region", "type", "runtime", "created_by", "status"],
+    "service_filter_params": ["service", "domain", "region", "type", "runtime", "created_by", "status", "deployment_accounts"],
```

Comment:
As we can not filter deployment_accounts  its better to remove it from service_filter_params list.

Reply:
Removed it from service_filter_params list